🤔
```
$ pylint --rcfile='tensorflow/tools/ci_build/pylintrc' 'tensorflow/compiler/xla/python/xla_client.py'
************* Module python.xla_client
tensorflow/compiler/xla/python/xla_client.py:66:0: W9015: "distributed_client, node_id" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:66:0: W9016: "distributed_client, node_id" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:125:6: R1720: Unnecessary "else" after "raise" (no-else-raise)
tensorflow/compiler/xla/python/xla_client.py:137:0: W9016: "name" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:137:0: W9012: Missing return type documentation (missing-return-type-doc)
tensorflow/compiler/xla/python/xla_client.py:137:0: W9006: "RuntimeError" not documented as being raised (missing-raises-doc)
tensorflow/compiler/xla/python/xla_client.py:153:6: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
tensorflow/compiler/xla/python/xla_client.py:137:0: W9012: Missing return type documentation (missing-return-type-doc)
tensorflow/compiler/xla/python/xla_client.py:159:0: R0205: Class 'OpMetadata' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:159:0: W9015: "op_name, op_type, source_file, source_line" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:159:0: W9016: "op_name, op_type, source_file, source_line" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:170:0: W9015: "op_name, op_type, skip_frames" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:170:0: W9016: "op_name, op_type, skip_frames" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:213:0: W9015: "dtype" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:213:0: W9016: "dtype" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:279:0: W9015: "pyval" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:279:0: W9016: "pyval" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:283:4: R1705: Unnecessary "else" after "return" (no-else-return)
tensorflow/compiler/xla/python/xla_client.py:341:0: W9015: "arguments, backend, executable" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:341:0: W9016: "arguments, backend, executable" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:352:0: W9016: "arguments, backend, executable" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:352:0: W9012: Missing return type documentation (missing-return-type-doc)
tensorflow/compiler/xla/python/xla_client.py:386:0: W9015: "lhs_dims, padding_type, rhs_dims, window_strides" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:386:0: W9016: "lhs_dims, padding_type, rhs_dims, window_strides" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:402:2: R1705: Unnecessary "elif" after "return" (no-else-return)
tensorflow/compiler/xla/python/xla_client.py:426:0: W9016: "fn, name, platform" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:441:0: R0205: Class 'PaddingConfigDimension' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:451:0: R0205: Class 'PaddingConfig' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:472:5: R1701: Consider merging these isinstance calls to isinstance(padding_config, (list, tuple)) (consider-merging-isinstance)
tensorflow/compiler/xla/python/xla_client.py:484:0: R0205: Class 'DotDimensionNumbers' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:512:2: R1705: Unnecessary "else" after "return" (no-else-return)
tensorflow/compiler/xla/python/xla_client.py:524:0: R0205: Class 'ConvolutionDimensionNumbers' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:608:0: R0205: Class 'OpSharding' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:623:0: R0205: Class 'PrecisionConfig' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:633:0: R0205: Class 'GatherDimensionNumbers' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:645:0: R0205: Class 'ScatterDimensionNumbers' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:657:0: R0205: Class 'ReplicaGroup' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
tensorflow/compiler/xla/python/xla_client.py:686:0: W9015: "enabled" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:686:0: W9016: "enabled" missing in parameter type documentation (missing-type-doc)
tensorflow/compiler/xla/python/xla_client.py:696:0: W9015: "client" missing in parameter documentation (missing-param-doc)
tensorflow/compiler/xla/python/xla_client.py:33:0: C0411: third party import "from absl import logging" should be placed before "from . import xla_extension as _xla" (wrong-import-order)
tensorflow/compiler/xla/python/xla_client.py:34:0: C0411: third party import "import numpy as np" should be placed before "from . import xla_extension as _xla" (wrong-import-order)

------------------------------------------------------------------
Your code has been rated at 8.47/10 (previous run: 8.47/10, +0.00)
```

--- a/tensorflow/tools/ci_build/ci_sanity.sh
+++ b/tensorflow/tools/ci_build/ci_sanity.sh
@@ -210,7 +210,8 @@ do_pylint() {
   # C0326 bad-whitespace
   # W0611 unused-import
   # W0622 redefined-builtin
-  grep -E '(\[E|\[W0311|\[W0312|\[C0330|\[C0301|\[C0326|\[W0611|\[W0622)' ${OUTPUT_FILE} > ${ERRORS_FILE}
+  # W9015 missing-param-doc
+  grep -E '(\[E|\[W0311|\[W0312|\[C0330|\[C0301|\[C0326|\[W0611|\[W0622|\[W9015)' ${OUTPUT_FILE} > ${ERRORS_FILE}
 
   N_ERRORS=0
   while read -r LINE; do

--- a/tensorflow/tools/ci_build/pylintrc
+++ b/tensorflow/tools/ci_build/pylintrc
@@ -1,7 +1,7 @@
 [MASTER]
 
 # Specify a configuration file.
-#rcfile=
+rcfile=tensorflow/tools/ci_build/pylintrc
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().

--- a/tensorflow/tools/ci_build/pylintrc
+++ b/tensorflow/tools/ci_build/pylintrc
@@ -1,7 +1,7 @@
 [MASTER]
 
 # Specify a configuration file.
-rcfile=tensorflow/tools/ci_build/pylintrc
+#rcfile=
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().

